app/src/main/java/com/greenfox/gitinder/model/MessageWrapper.java

```diff
@@ -0,0 +1,18 @@
+package com.greenfox.gitinder.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MessageWrapper implements Serializable {
```

Comment:
@kolekd if you pass the match object through the intent, you dont need this class